src/Blog/templates/post.phtml

```diff
@@ -1,4 +1,12 @@
 <?php
+
+declare(strict_types=1);
+
+/**
+ * @var League\Plates\Template\Template $this
```

Comment:
Isn't this our ViewRenderer?

Reply:
Unfortunately not, because the plugins / extensions are not listed in Plates.

https://github.com/laminas/laminas-view/blob/50ae82af3e2da96108490ceb474480e6df226993/src/Renderer/PhpRenderer.php#L46-L89

Reply:
I confused this repo with a general-purpose example: my bad :-)

Reply:
No, no, your intention is correct. It _should be_ a general and good example.
But there is still a lot of work missing. :stuck_out_tongue_winking_eye: